packages/unit-testing/stubs/currentRecord/CurrentRecordInstance.js

```diff
@@ -0,0 +1,463 @@
+define(['./Sublist', './Field'], function (Sublist, Field) {
+    function CurrentRecord() {
```

Comment:
I think properties are missing (id, isDynamic, type) the three of them are read-olin

Reply:
Done